app/models/distribution_point.rb

```diff
@@ -39,13 +39,13 @@ class DistributionPoint < ApplicationRecord
   end
 
   scope :outdated, ->(timing = 4.hours.ago) { where("updated_at < ?", timing) }
-  scope :inactive, -> { unscope(:where).where(active: false) }
+  scope :archived, -> { unscope(:where).where(archived: true) }
 
   geocoded_by :address
 
   def self.to_csv
     attributes = %w[
-      id updated_at active facility_name address city county state latitude longitude
+      id updated_at active facility_name address city county state latitude longitude archived
```

Comment:
I'm thinking the CSV export would never include archived distribution points, so do we really need that column here?